packages/libraries/cli/src/commands/schema/fetch.ts

```diff
@@ -59,7 +59,7 @@ const LatestSchemaVersionQuery = graphql(/* GraphQL */ `
   ) {
     latestValidVersion(target: $target) {
       id
-      valid
+      isValid
```

Comment:
I see that `valid` exists still in the schema, my eyes are working right?

Reply:
yup its still used in previous versions of the cli